### PR TITLE
build vocabulary from sentences

### PR DESCRIPTION
**Pull request by SoluMilken**
Please merge #10 before merging this one.
This is because flake8  does not work in this pr.

### CODE REVIEW

File: strpipe/toolkit/build_vocabulary.pyx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import operator
+
+
+def build_vocabulary_from_sentences(
+        sentences: list[list[str]],
+        vocab_size: int = 1e+8,
+        sos_token: str = '<SOS>',
+        eos_token: str = '<EOS>',
+        unk_token: str = '<UNK>',
+        pad_token: str = '<PAD>',
```
Comment (stegben) [inline strpipe/toolkit/build_vocabulary.pyx]: 這些 default TOKEN 要集中起來管理

File: strpipe/toolkit/build_vocabulary.pyx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        str unk_token,
+        str pad_token,
+    ):
+    cdef set default_tokens = set(
+        [
+            sos_token, eos_token,
+            unk_token, pad_token,
+        ],
+    )
+    cdef unsigned int max_len = 0
```
Comment (stegben) [inline strpipe/toolkit/build_vocabulary.pyx]: 這個變數沒用到？

File: strpipe/toolkit/build_vocabulary.pyx (hunk 1 above)
Comment (SoluMilken) [reply strpipe/toolkit/build_vocabulary.pyx]: 好喔

Comment (SoluMilken) [reply strpipe/toolkit/build_vocabulary.pyx]: 0ac0fbc

File: strpipe/toolkit/build_vocabulary.pyx (hunk 2 above)
Comment (SoluMilken) [reply strpipe/toolkit/build_vocabulary.pyx]: b337405

File: strpipe/toolkit/default_tokens.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+class DefaultTokens:
+
+    sos_token = '<SOS>'
+    eos_token = '<EOS>'
+    unk_token = '<UNK>'
+    pad_token = '<PAD>'
```
Comment (stegben) [inline strpipe/toolkit/default_tokens.py]: class 名稱已經有 Token 了，attributes 應該就不需要 pad `_token` 嘛？

Comment (SoluMilken) [reply strpipe/toolkit/default_tokens.py]: rrrrrrr

Comment (SoluMilken) [reply strpipe/toolkit/default_tokens.py]: 8a53fc1

**Pull request merged**